sdk/avalon_sdk/connector/blockchains/ethereum/ethereum_worker_registry.py

```diff
@@ -217,18 +221,18 @@ def worker_register(self, worker_id, worker_type, organization_id,
         """
 
         if (self.__contract_instance is not None):
-            if not is_valid_hex_str(worker_id):
+            if not is_valid_hex_of_length(worker_id, 32):
                 logging.error("Invalid worker id {}".format(worker_id))
                 return None
```

Comment:
Return ContractResponse.ERROR instead of None for all invalid inputs if it is write transaction to chain.